src/instruction-handlers.ts

```diff
@@ -79,9 +80,27 @@ export const goto = async (instruction: GoTo): Promise<void> => {
   const position = new Position(line - 1, column - 1);
   editor.selection = new Selection(position, position);
   editor.revealRange(editor.selection);
-  await wait(getPause());
+  await timeout(getPause());
 };
 
-function wait(time: number) {
+export const wait = (instruction: Wait): Promise<void> => {
+  return new Promise(async (resolve, reject) => {
+    if (typeof instruction.length === 'number') {
+      await timeout(instruction.length);
+      await timeout(getPause());
```

Comment:
Why the `await timeout(getPause());`? Seems the first await should be enough as that is what is configured.